packages/react/src/River/River.module.css

```
@@ -22,6 +22,11 @@
   object-fit: cover;
 }
 
+.River__visual--rounded,
+.River__visual--rounded img {
+  border-radius: var(--brand-borderRadius-large);
+}
```

another would be to apply it to the visual area + use overflow hidden

```suggestion
.River__visual--rounded {
  overflow: hidden;
  border-radius: var(--brand-borderRadius-large);
}
```

---

I like this suggestion, handles more use-cases, while separating concerns in the parent 👍. Going with this approach.